src/lib/pubkey/pqcrystals/info.txt

```diff
@@ -0,0 +1,18 @@
+<defines>
+PQCRYSTALS -> 20240228
+</defines>
+
+<module_info>
+name -> "CRYSTALS"
+brief -> "Base utilities for CRYSTALS-Kyber/ML-KEM and CRYSTALS-Dilithium/ML-DSA. CRYptographic SuiTe for Algebraic LatticeS"
+type -> "Internal"
+</module_info>
+
+<requires>
+</requires>
```

Comment:
`xof`

Reply:
Mhh, actually this requires `xof` only when used in conjunction with Diltihium. Kyber doesn't unpack from a XOF output.

Reply:
I see you added ifdef guard, that's fine too, just has to be able to compile one way or another